src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,255 @@
+import { stylesheet, docsIntro } from "../../../.storybook/utils.js";
+import { ALERT_SIZE, ALERT_TYPE, ALERT_ICON } from "./enums";
+
+export default {
+  title: "custom-elements/vl-alert",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  parameters: {
+    docs: {
+      description: {
+        component: docsIntro({
+          root: "alert",
+          intro:
+            "Use the alert (or notice) component to keep the user updated about important information.",
+        }),
+      },
+    },
+  },
+  args: {
+    title: "Opgelet!",
+    icon: "warning",
+    size: "normal",
+    type: "info",
+    closable: false,
+    buttonText: "Fout melden",
+    slotTitle: "Alert titel via slot",
+    content: "U heeft geen rechten om deze actie uit te voeren.",
+  },
+  argTypes: {
+    title: {
+      name: "data-vl-title",
+      type: { summary: "string" },
+      description: "Attribuut wordt gebruikt om de titel te bepalen.",
+      table: {
+        defaultValue: { summary: "" },
+      },
+    },
+    icon: {
+      name: "data-vl-icon",
+      type: {
+        summary: `${ALERT_ICON.WARNING} | ${ALERT_ICON.CHECK} | ${ALERT_ICON.INFO_CIRCLE}`,
+      },
+      description: "Attribuut wordt gebruikt om het icoon type te bepalen.",
+      control: {
+        type: "select",
+        options: [ALERT_ICON.WARNING, ALERT_ICON.CHECK, ALERT_ICON.INFO_CIRCLE],
+      },
+      table: {
+        defaultValue: { summary: `"${ALERT_ICON.WARNING}"` },
+      },
+    },
+    size: {
+      name: "data-vl-size",
+      type: {
+        summary: `${ALERT_SIZE.NORMAL} | ${ALERT_SIZE.SMALL}`,
+      },
+      description:
+        "Attribuut activeert een variant van de waarschuwing maar kleiner.",
+      control: {
+        type: "select",
+        options: [ALERT_SIZE.NORMAL, ALERT_SIZE.SMALL],
+      },
+      table: {
+        defaultValue: { summary: `"${ALERT_SIZE.NORMAL}"` },
+      },
+    },
+    type: {
+      name: "data-vl-type",
+      type: {
+        summary: `${ALERT_TYPE.INFO} | ${ALERT_TYPE.SUCCESS} | ${ALERT_TYPE.WARNING} | ${ALERT_TYPE.ERROR}`,
+      },
+      description:
+        "Attribuut bepaalt de soort van waarschuwing, foutmelding, probleemmelding of succesmelding.",
+      control: {
+        type: "select",
+        options: [
+          ALERT_TYPE.INFO,
+          ALERT_TYPE.SUCCESS,
+          ALERT_TYPE.WARNING,
+          ALERT_TYPE.ERROR,
+        ],
+      },
+      table: {
+        defaultValue: { summary: `"${ALERT_TYPE.INFO}"` },
+      },
+    },
+    closable: {
+      name: "data-vl-closable",
+      type: { summary: "boolean" },
+      description:
+        "Attribuut wordt gebruikt om de optie toe te voegen om de waarschuwing te sluiten door op het sluit icoon te klikken in de rechterbovenhoek.",
+      table: {
+        defaultValue: { summary: "false" },
+      },
+    },
+    content: {
+      name: "content (for demo purposes)",
+      description: "Test",
+    },
+    slotTitle: {
```

Comment:
disable de controls van properties als die niet gebruikt worden in stories

Reply:
Done.